heft-plugins/heft-sass-plugin/package.json

```diff
@@ -8,14 +8,12 @@
     "directory": "heft-plugins/heft-sass-plugin"
   },
   "homepage": "https://rushstack.io/pages/heft/overview/",
-  "main": "lib/index.js",
```

Comment:
I don't disagree with this. It may not be required by plugins in general, but would be good to have a common interface/style that we use for our own internal plugins. I can look into implementing that on the larger PR with all the other plugins.